executor/pipeline/pipeline.rs

```diff
@@ -186,15 +193,14 @@ impl<Snapshot: WritableSnapshot + 'static> Pipeline<Snapshot, WritePipelineStage
         let output_variable_positions = executable_stages.last().unwrap().output_row_mapping();
         let context = ExecutionContext::new(Arc::new(snapshot), thing_manager, parameters);
         let mut last_stage = WritePipelineStage::Initial(Box::new(InitialStage::new_empty(context)));
+        // TODO: Receive as arg?
+        let executable_functions = Arc::new(ExecutableFunctionRegistry::empty());
```

Comment:
Not sure if we should receive it from somewhere?

Reply:
Yeah krishnan should be wiring this through in his PRs at some point!